examples/site.yml

```diff
@@ -9,8 +9,5 @@
     - { role: "azavea.docker" }
 
   tasks:
-    - name: Bring up a test container
```

Comment:
Is there a reason this was taken out? I agree that it probably has little value, but given that, it at least exercises the interaction between Ansible and the Docker Engine (which happens via the Docker for Python SDK).